app/controllers/admin/staffings_controller.rb

```diff
@@ -135,8 +135,8 @@ def create
         # Assumes that a staffing is shorter than 24 hours. This means that if end_time > start_time, it assumes it ends on the same day. 
         # If end_time < start_time, it ends the next day.
 
-        staffing.start_time = DateTime.parse(start_time)
```

Comment:
Is this the fix for the 1h off issue?

Reply:
Yes, it is! Just letting Rails parse it seems to fix it.